script.js

```diff
@@ -0,0 +1,133 @@
+const suits = ["Hearts", "Spades", "Diamonds", "Clubs"];
+const cardValues = [
+  "Ace",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "10",
+  "Jack",
+  "Queen",
+  "King",
+];
+
+const initialDeck = [];
+let currentDeck = [];
+
+const buildDeck = () => {
+  suits.forEach((suit) => {
+    cardValues.forEach((cardValue) => {
+      let card = {
+        suit,
+        cardValue,
+      };
+      initialDeck.push(card);
+    });
+  });
+};
+
+buildDeck();
+
+divCards = document.getElementById("cards");
+
+const createNodeList = (list) => {
+  return list.map((card) => {
+    const el = document.createElement("div");
+    el.textContent = `${card.cardValue} of  ${card.suit}`;
+    return el;
+  });
+};
+
+// Button Initial Deck
+const btnInitialDeck = document.querySelector("#initial-deck-button");
+btnInitialDeck.addEventListener("click", showInitialDeck);
+
+function showInitialDeck() {
+  const nodeList = createNodeList(initialDeck);
+  divCards.replaceChildren(...nodeList);
+  currentDeck = [...initialDeck];
+}
```

Comment:
This function share a lot simialirites with `showShuffleDeck` maybe you could consider refactoring it!